schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -1874,10 +1877,87 @@
       "then": {
         "required": [
           "type",
-          "velero"
+          "velero",
+          "etcdBackup"
         ]
       }
     },
+    "Spec.Distribution.Modules.Dr.EtcdBackup": {
+      "type": "object",
+      "description": "Configuration for the ETCD backup package.",
+      "additionalProperties": false,
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "s3",
+            "pvc",
+            "none",
+            "all"
+          ],
+          "description": "The type of the etcd backup to enable, options are:\n- `none`: no etcd backup CronJob will be installed and no etcd backup will be performed.\n- `s3`: the etcd-backup-s3 package will be enabled. It will deploy a CronJob which continuously snapshots a healthy etcd node and will save the backups in a configured S3 bucket.\n- `pvc`: the etcd-backup-pvc package will be enabled. It will deploy a CronJob which continuously snapshots a healthy etcd node and will save the backups in a configured PersistentVolumeClaim.\n- `all`: both kinds of backups will be enabled.\n\nDefault is `none`."
+        },
+        "backupPrefix": {
+          "type": "string",
+          "additionalProperties": false,
+          "description": "A prefix to be prepended to the backup filenames. If unset, the prefix defaults to the cluster's name."
+        },
+        "persistentVolumeClaim": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Configuration for the PersistentVolumeClaim where `etcd-backup-pvc` will store its backups.",
+          "properties": {
+            "schedule": {
+              "type": "string",
+              "description": "The cron expression for the `etcd-backup-pvc` backup schedule. Default is `0 1 * * *`."
+            },
+            "claimName": {
+              "type": "string",
+              "description": "The PersistentVolumeClaim name where the backup will be saved."
+            },
+            "retentionTime": {
+              "type": "string",
+              "description": "The retention time of the backups inside the PersistentVolumeClaim. Follows rclone's `min-age` format. Example: '30d' for 30 days."
+            }
+          }
+        },
+        "s3": {
```

Comment:
there's missing here the required properties definition, for a user to use the s3 type, they must set all fields but schedule, retentionTime and insecure that can be left to their default values or changed if wanted.